actions/views/rhs.ts

```diff
@@ -292,22 +329,24 @@ export function selectPost(post) {
     };
 }
 
-export function selectPostCard(post) {
+export function selectPostCard(post: Post) {
     return {type: ActionTypes.SELECT_POST_CARD, postId: post.id, channelId: post.channel_id};
 }
 
 export function openRHSSearch() {
-    return (dispatch) => {
+    return (dispatch: DispatchFunc) => {
         dispatch(clearSearch());
         dispatch(updateSearchTerms(''));
         dispatch(updateSearchResultsTerms(''));
 
         dispatch(updateRhsState(RHSStates.SEARCH));
+
+        return {data: true};
     };
 }
 
-export function openAtPrevious(previous) {
-    return (dispatch, getState) => {
+export function openAtPrevious(previous: any) { // TODO Could not find the proper type. Seems to be in several props around
```

Comment:
Looks like we'll have to define this type somewhere. Don't have to do it now, but eventually.